Automattic/woocommerce-payments
# Remove total from active dispute task title when from multiple currencies

Fixes #6613

#### Changes proposed in this Pull Request





This PR removes the calculated total amounts from the active dispute task title when disputes are for multiple currencies. This simplifies the readability of the title.

#### WC Home

**Before**
<img width="684" alt="Screenshot 2023-07-11 at 10 56 00" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/c65ed449-4771-4a56-9c65-ccbe982d020e">

**After** (ignore mouse-hover grey colouring)
<img width="683" alt="Screenshot 2023-07-11 at 10 48 40" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/f7867329-395a-4992-bb96-af641f62d3dd">


#### Payments Overview

**Before**

![baaa7426591246174520e61d98635774](https://github.com/Automattic/woocommerce-payments/assets/3147296/b22bafbe-2464-423d-a19e-7a83324c4751)

**After**

![4c0675d480b7807ff543613a02a096f9](https://github.com/Automattic/woocommerce-payments/assets/3147296/a6a11dde-b17e-4a32-ac2f-2f1a2285dd9e)






#### Testing instructions





* Using a multi-currency WCPay store, create disputed transactions across multiple currencies. These should be due within 7 days to view the active dispute task. 


**Mocking disputes**

**Payments Overview:** Add the following in `client/overview/index.js`, replacing `const { disputes: activeDisputes } =`, modifying `due_by` if necessary:

```js
const activeDisputes = [
	{
		wcpay_disputes_cache_id: 1,
		stripe_account_id: 'stripe_account_id_1',
		dispute_id: 'dispute_id_1',
		charge_id: 'charge_id_1',
		amount: 100,
		currency: 'USD',
		reason: 'DisputeReason',
		source: 'source_1',
		order_number: 123456,
		order: null,
		customer_name: 'John Doe',
		customer_email: 'john.doe@example.com',
		customer_country: 'USA',
		status: 'DisputeStatus',
		created: '2023-07-01',
		due_by: '2023-07-15',
	},
	{
		wcpay_disputes_cache_id: 2,
		stripe_account_id: 'stripe_account_id_2',
		dispute_id: 'dispute_id_2',
		charge_id: 'charge_id_2',
		amount: 200,
		currency: 'EUR',
		reason: 'DisputeReason',
		source: 'source_2',
		order_number: 654321,
		order: null,
		customer_name: 'Jane Smith',
		customer_email: 'jane.smith@example.com',
		customer_country: 'UK',
		status: 'DisputeStatus',
		created: '2023-07-02',
		due_by: '2023-07-16',
	},
	{
		wcpay_disputes_cache_id: 3,
		stripe_account_id: 'stripe_account_id_3',
		dispute_id: 'dispute_id_3',
		charge_id: 'charge_id_3',
		amount: 300,
		currency: 'JPY',
		reason: 'DisputeReason',
		source: 'source_3',
		order_number: 987654,
		order: null,
		customer_name: 'David Lee',
		customer_email: 'david.lee@example.com',
		customer_country: 'Japan',
		status: 'DisputeStatus',
		created: '2023-07-03',
		due_by: '2023-07-17',
	},
];
```

**WC Home**: return the following in the function `get_disputes_needing_response()` from `includes/admin/tasks/class-wc-payments-task-disputes.php`:

```php
// mock disputes with multiple currencies.
return [
	[
		'dispute_id' => 'dp_123',
		'amount'     => 10000,
		'currency'   => 'USD',
		'due_by'     => ( new \DateTime( 'now', new \DateTimeZone( 'UTC' ) ) )->modify( '+1 day' )->format( 'Y-m-d H:i:s' ),
	],
	[
		'dispute_id' => 'dp_456',
		'amount'     => 10000,
		'currency'   => 'EUR',
		'due_by'     => ( new \DateTime( 'now', new \DateTimeZone( 'UTC' ) ) )->modify( '+2 day' )->format( 'Y-m-d H:i:s' ),
	],
	[
		'dispute_id' => 'dp_789',
		'amount'     => '',
		'currency'   => 'USD',
		'due_by'     => ( new \DateTime( 'now', new \DateTimeZone( 'UTC' ) ) )->modify( '+3 day' )->format( 'Y-m-d H:i:s' ),
	],
];
```

-------------------


**Post merge**
